atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/SweepCommand.java

```diff
@@ -0,0 +1,176 @@
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.cli;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicLong;
+
+import com.google.common.base.Functions;
+import com.google.common.base.Optional;
+import com.google.common.base.Predicates;
+import com.google.common.base.Stopwatch;
+import com.google.common.collect.Maps;
+import com.google.common.collect.Sets;
+import com.google.common.io.BaseEncoding;
+import com.palantir.atlasdb.AtlasDbConstants;
+import com.palantir.atlasdb.cli.api.AtlasDbServices;
+import com.palantir.atlasdb.cli.api.SingleBackendCommand;
+import com.palantir.atlasdb.keyvalue.api.SweepResults;
+import com.palantir.atlasdb.schema.generated.SweepPriorityTable;
+import com.palantir.atlasdb.schema.generated.SweepTableFactory;
+import com.palantir.atlasdb.sweep.SweepTaskRunner;
+import com.palantir.atlasdb.transaction.api.Transaction;
+import com.palantir.atlasdb.transaction.impl.TxTask;
+import com.palantir.common.base.Throwables;
+
+import io.airlift.airline.Command;
+import io.airlift.airline.Option;
+
+@Command(name = "sweep", description = "Sweep old table rows")
+public class SweepCommand extends SingleBackendCommand {
+
+    @Option(name = {"-n", "--namespace"},
+            description = "An atlas namespace to sweep")
+    String namespace;
+
+    @Option(name = {"-t", "--table"},
+            description = "An atlas table to sweep")
+    String table;
+
+    @Option(name = {"-r", "--row"},
+            description = "A row to start from (hex encoded bytes)")
+    String row;
+
+    @Option(name = {"-a", "--all"},
+            description = "Sweep all tables")
+    boolean sweepAllTables;
+
+    @Option(name = {"-b", "--batch"},
+            description = "Sweep batch size (default: 2000)")
+    int sweepBatchSize = 2000;
+
+    @Option(name = {"-p", "--pause"},
+    description = "The number of milliseconds to pause between each batch of deletes (default: 5000)")
+    long sweepPauseMillis = 5000;
+
+	@Override
+	protected int execute(final AtlasDbServices services) {
+        SweepTaskRunner sweepRunner = services.getSweepTaskRunner();
+
+        if (!((namespace != null) ^ (table != null) ^ sweepAllTables)) {
+            System.err.println("Specify one of --namespace, --table, or --all options.");
+            return 1;
+        }
+        if ((namespace != null) && (row != null)) {
+            System.err.println("Specifying a start row option doesn't really make sense specified against multiple tables.");
```

Comment:
wording seems off, suggest

`"Cannot specify start row " + row + " sweeping multiple tables in namespace " + namespace`